cpu/kinetis/Makefile.include

```diff
@@ -27,7 +27,8 @@ LINKER_SCRIPT = kinetis.ld
 
 ROM_START_ADDR = 0x00000000
 RAM_BASE_ADDR  = 0x20000000
-RAM_START_ADDR = $$(($(RAM_BASE_ADDR)-($(KINETIS_SRAM_L_SIZE) * 1024)))
+RAM_START_ADDR := $(shell printf "0x%08x" $$(($(RAM_BASE_ADDR)-($(KINETIS_SRAM_L_SIZE) * 1024))))
```

Comment:
Did you use `:=` so it is only evaluated once?

Reply:
yes!